go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/web-platform-tests/wpt.fyi
 
-go 1.21.13
```

Comment:
I am curious if go 1.22 will allow us to keep go1.12.13 here for now (but keep the rest of the changes) Then in the next PR, we could bump this. It might be something to try.

Reply:
I tried it, but the module updates in this PR require 1.22, so checks fail demanding updates to go.mod. 

Maybe it could work if we split the runtime and module updates in two separate PRs. I can try that in 1.23, once it is available in AppEngine.